plugin-server/src/kafka/batch-consumer.ts

```diff
@@ -322,7 +331,7 @@ export const startBatchConsumer = async ({
     const isHealthy = () => {
         // We define health as the last consumer loop having run in the last
         // minute. This might not be bullet-proof, let's see.
-        return Date.now() - lastHeartbeatTime < 60000
```

Comment:
snuck this change in
the broker will allow us 90_000 between heartbeats, we allow ourselves 60_000
we sometimes see processing take greater than 60 seconds
no bueno, 
we could declare ourselves unhealthy when we're just slow